azkaban-execserver/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -651,6 +652,11 @@ private void prepareJobProperties(ExecutableNode node) throws IOException {
     node.setInputProps(props);
   }
 
+  private void customizeJobProperties(Props props) {
```

Comment:
do we want to name the function a bit better so that we know what exactly is being "customized"?
maybe insertMemoryCheckToJobProperties (or something)

otherwies commenting the method would also help